fe/fe-core/src/main/java/com/starrocks/sql/parser/StarRocks.g4

```diff
@@ -14,14 +14,16 @@ singleStatement
 statement
     : queryStatement                                                                    #statementDefault
     | EXPLAIN (LOGICAL | VERBOSE | COSTS)? queryStatement                               #explain
-    | EXPLAIN? INSERT INTO qualifiedName (WITH LABEL lable=identifier)? columnAliases?
+    | EXPLAIN? INSERT INTO qualifiedName (WITH LABEL label=identifier)? columnAliases?
     (queryStatement | (VALUES expressionsWithDefault (',' expressionsWithDefault)*))    #insert
     | CREATE TABLE (IF NOT EXISTS)? qualifiedName
         ('(' identifier (',' identifier)* ')')? comment?
         partitionDesc?
         distributionDesc?
         properties?
         AS queryStatement                                                               #createTableAsSelect
+    | (EXPLAIN (LOGICAL | VERBOSE | COSTS)?)?
```

Comment:
Insert Statement does not support other types of explain, I think Update should be consistent?

Reply:
How about all support those explain types? If it is useful in select, it is also useful in "insert xxx select"

Reply:
Yes, just ensure that the insert and update logic are consistent.